axelrod/tests/unit/test_version.py

```diff
@@ -0,0 +1,11 @@
+"""
+Tests the version number
+"""
+from axelrod import __version__
+import unittest
+
+expected_version = "1.11.0"
+
+class TestVersion(unittest.TestCase):
+    def test_version(self):
+        self.assertEqual(__version__, expected_version)
```

Comment:
I don't really see the point of this test. It's just testing that Python correctly returns the value of an attribute, not any logic that we have coded into the library.